src/utils/Balance.tsx

```diff
@@ -158,8 +158,8 @@ const useBalanceHook = () => {
     if (!tokenPrices?.helium) return
 
     const heliumPrice = tokenPrices.helium[currency.toLowerCase()]
-
-    return new Balance(heliumPrice, CurrencyType.usd)
+    // Multiple by 10 the 8th power to get the price in USD
+    return new Balance(heliumPrice * 10 ** 8, CurrencyType.usd)
```

Comment:
I wonder if `Balance.fromFloatAndTicker(heliumPrice, 'USD')` would work here?

Reply:
Ahh let me try that

Reply:
It worked 🎉